tsconfig.json

```diff
@@ -1,14 +1,15 @@
 {
     "include": ["./src/**/*", "./src/custom.d.ts"],
-      "jsx": "react",
-      "compilerOptions": {
+    "jsx": "react",
+    "compilerOptions": {
         "baseUrl": "src",
         "jsx": "preserve",
         "esModuleInterop": true,
         "experimentalDecorators": true,
         "emitDecoratorMetadata": true,
         "noEmit": true,
         "skipLibCheck": true,
-        "moduleResolution": "node"
-      }
+        "moduleResolution": "node",
+        "lib": ["dom", "dom.iterable", "ES2021.String"]
```

Comment:
Any reason for adding these types for built-in JS APIs?

Reply:
> Any reason for adding these types for built-in JS APIs?

- Add `"lib": ["dom", "dom.iterable"]` to support `URLSearchParams(location.search)` in src/signup-success/index.tsx line 53.
- Add `"lib": ["ES2021.String"]` to support `.replaceAll` method in src/signup-success/index.tsx line 57.